src/components/GameView/GameOverDialog.vue

```diff
@@ -1,61 +1,61 @@
 <template>
-  <v-dialog v-model="show" persistent max-width="650">
-    <v-card id="game-over-dialog">
-      <div id="dialog-header">
-        <div v-if="currentMatch" class="d-flex">
-          <v-card-title :data-cy="headingDataAttr" class="mt-8">
-            <h1 class="dialog-header">{{ heading }}</h1>
-          </v-card-title>
-          <v-card-text class="d-flex justify-end mr-4 mt-4">
-            <v-img :src="logoSrc" :data-cy="logoDataAttr" class="logo-image-match" />
-          </v-card-text>
-        </div>
-        <div v-else>
-          <v-card-title :data-cy="headingDataAttr">
-            <h1 class="dialog-header">{{ heading }}</h1>
-          </v-card-title>
-          <v-card-text class="d-flex justify-center">
-            <v-img :src="logoSrc" :data-cy="logoDataAttr" class="logo-image" />
-          </v-card-text>
+  <base-dialog v-model="show" id="game-over-dialog">
+    <template #title>
+      <h1 :data-cy="headingDataAttr" class="dialog-header">{{ heading }}</h1>
+      <v-img v-if="currentMatch && !isMobilePortrait" :src="logoSrc" :data-cy="logoDataAttr" class="logo-image-match" />
+    </template>
```

Comment:
The base-dialog has a `title` prop for when the title is just text, and a `#title` slot for when the header should have additional content. Here we're using the slot (this is the use case that drove creating the slot in order to have the logo image appear in the upper-right corner

![image](https://user-images.githubusercontent.com/6520704/233676322-8d27d8ec-dd35-43bb-928d-ca850fc4d8ce.png)